communis-services/src/main/java/com/rhema/communis/member/MemberController.java

```diff
@@ -35,6 +35,12 @@ public MemberController(MemberService memberService){
                 memberService.create(person)), HttpStatus.OK);
     }
 
+    @GetMapping("")
+    public ResponseEntity<CommunisResponse> findAll(){
```

Comment:
shouldn't it be getAll? Because I guess find has the connotation that the client asked something to be searched? I'm new to this so I'm not sure about what I'm saying but just thinking out loud.

Reply:
Yeah that makes sense. I can do that.